backend/api/teams/actions.py

```diff
@@ -234,3 +236,75 @@ def post(self, team_id):
             error_msg = f"TeamMembers DELETE - unhandled error: {str(e)}"
             current_app.logger.critical(error_msg)
             return {"Error": error_msg}, 500
+
+
+class TeamsActionsMessageMembersAPI(Resource):
+    @tm.pm_only(True)
+    @token_auth.login_required
+    def post(self, team_id):
+        """
+        Message all team members
+        ---
+        tags:
+          - teams
+        produces:
+          - application/json
+        parameters:
+            - in: header
+              name: Authorization
+              description: Base64 encoded session token
+              required: true
+              type: string
+              default: Token sessionTokenHere==
+            - name: team_id
+              in: path
+              description: Unique team ID
+              required: true
+              type: integer
+              default: 1
+            - in: body
+              name: body
+              required: true
+              description: JSON object for creating message
+              schema:
+                properties:
+                    subject:
+                        type: string
+                        default: Thanks
+                        required: true
+                    message:
+                        type: string
+                        default: Thanks for your contribution
+                        required: true
+        responses:
+            200:
+                description: Message sent successfully
+            401:
+                description: Unauthorized - Invalid credentials
+            403:
+                description: Forbidden
+            500:
+                description: Internal Server Error
+        """
+        try:
+            authenticated_user_id = token_auth.current_user()
+            message_dto = MessageDTO(request.get_json())
+            message_dto.from_user_id = authenticated_user_id
+            message_dto.validate()
```

Comment:
We need to restrict only to team managers and the managers of the organization of team.